frame/treasury/src/lib.rs

```diff
@@ -54,25 +54,27 @@
 //! ## GenesisConfig
 //!
 //! The Treasury module depends on the [`GenesisConfig`](./struct.GenesisConfig.html).
-
 #![cfg_attr(not(feature = "std"), no_std)]
-
+#[cfg(test)]
+mod tests;
```

Comment:
Should I move `tests` back to `lib.rs`? Move it outside is not friendly for reviewing this pr.